teamcity/flake8_plugin.py

```diff
@@ -1,74 +1,6 @@
 try:
-    import pycodestyle as pep8
+    from flake8.formatting import base  # noqa
 except ImportError:
-    import pep8
-
-import re
-
-from teamcity.messages import TeamcityServiceMessages
-from teamcity import __version__, is_running_under_teamcity
-
-
-name = 'teamcity'
-version = __version__
-enable_teamcity = is_running_under_teamcity()
```

Comment:
@shalupov Actually, I don't think that the `--teamcity` option is needed for the V3 plugin to work. See https://github.com/JetBrains/teamcity-messages/pull/106/files#diff-1f0f9bd5f527eabbc8fe0ef02be698e0R21

Basically, if you look closely, you'll see that I added a `--teamcity` option but I don't check whether `enable_teamcity` is set in the `format` method.

The `--teamcity` option is actually a no-op. I only include it in the V3 plugin so that if existing build scripts are passing it, the plugin won't complain that `--teamcity` is an illegal option.

However, currently one does need to pass `--format=teamcity-messages` to activate this plugin, which means folks have to change their scripts in order to activate it, which is unfortunate.